uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/context/GridBodyCellEditContext.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.uberfire.ext.wires.core.grids.client.widget.context;
+
+import java.util.Optional;
+
+import com.ait.lienzo.client.core.shape.wires.handlers.MouseEvent;
```

Comment:
This import is used just in javadoc, not sure if it is usual practice. What do you think?

Reply:
Well, IDEA added it for me.. and removing it gives an error in IDEA.. so prefer to keep it TBH.

Reply:
I am fine with it.